Assets/AppManager.cs

```diff
@@ -79,10 +80,75 @@ void Awake()
 
         void Start()
         {
-            exmgr.CollectDefination(cfgmgr.config.ExDir);
-            exsmgr.CollectDefination(cfgmgr.config.ExSessionDir);
+             if (ui == null)
```

Comment:
no need to check these references, when we drag connect them in editor, it can't be null.